api/src/main/java/org/openmrs/module/mchapp/db/hibernate/HibernateImmunizationCommoditiesDAO.java

```diff
@@ -6,22 +6,18 @@
 import org.apache.commons.logging.LogFactory;
 import org.hibernate.Criteria;
 import org.hibernate.SQLQuery;
-import org.hibernate.Session;
 import org.hibernate.SessionFactory;
 import org.hibernate.cfg.NotYetImplementedException;
+import org.hibernate.classic.Session;
```

Comment:
What is this import for?

Reply:
Why not use this **import org.hibernate.Session** instead?

Reply:
[An extension of the Session API, including all deprecated methods from Hibernate2. This interface is provided to allow easier migration of existing applications](https://docs.jboss.org/hibernate/core/3.3/api/org/hibernate/classic/Session.html). I agree our new code should use org.hibernate.Session.